docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+
+  database:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_PASSWORD=development
+
+  app:
+    image: github-inbox
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - 3000:3000
```

Comment:
I'm leaning toward leaving it as is to that things will work for a wider audience, but maybe @andrew can weigh in.

(I did not know about Docker Compose 1.9, or the update to this behavior in the Docker Compose YAML format, so thanks for the heads up about it.)